lib/ZSubs.c

```diff
@@ -93,6 +93,12 @@ ZSubscriptions(register ZSubscription_t *sublist,
     int size, start, numok;
     Z_AuthProc cert_routine;
 
+    if (ZGetFD() < 0)
```

Comment:
Please add {} to all new ifs. braceless if statements should be discouraged.

Reply:
This was copied and pasted verbatim from a different file that didn't have curly braces. I added them.